db/migrate/20190508150729_add_start_date_end_date_pet_picture_url_to_listings.rb

```diff
@@ -0,0 +1,7 @@
+class AddStartDateEndDatePetPictureUrlToListings < ActiveRecord::Migration[5.2]
+  def change
+    add_column :listings, :start_date, :date
+    add_column :listings, :end_date, :date
+    add_column :listings, :pet_picture, :string
+  end
+end
```

Comment:
Is there a reason why this became four migration files (2x removing and 2x adding)? Maybe you tested with one file first to see it worked or?

Reply:
it's a learning experience man :P 
we tested with Remove...From + Add...To

Reply:
Always great stuff with great learning experiences!!!